src/net/sourceforge/kolmafia/KoLCharacter.java

```diff
@@ -5377,7 +5380,7 @@ record StatExp(DoubleModifier exp, DoubleModifier pct) {}
           };
       Function<DoubleModifier, Double> calc =
           (DoubleModifier statPct) ->
-              (finalBaseExp + finalExp) * (1 + newModifiers.get(statPct) / 100.0f);
+              (finalBaseExp + finalExp) * (1 + newModifiers.getDouble(statPct) / 100.0f);
```

Comment:
I realize this is modified existing code, but, 100.0f? As in a single float?

Reply:
To clarify: the change you want is to remove the `f`?

Reply:
There are more `f` in the stat lines ~5390 -- do you want them gone as well?

Reply:
Yes. when you are calculating a double, why force some of the factors to be simple floats?